src/cpp/common/tls_credentials_options.cc

```diff
@@ -33,7 +31,7 @@ void TlsKeyMaterialsConfig::set_pem_root_certs(grpc::string pem_root_certs) {
 
 void TlsKeyMaterialsConfig::add_pem_key_cert_pair(
```

Comment:
It seems that the ownership of key materials that get passed in to `TlsKeyMaterialsConfig` and `TlsCredentialReloadArg` are treated differently. In the former, the ownership is transferred while in the latter the ownership is not transferred. Since in both cases the key materials are input from users, should not we have a consistent behavior?

Reply:
For consistency, ownership will not be transferred in either case.